requirements_all.txt

```diff
@@ -64,6 +64,9 @@ cherrypy==8.1.0
 coinmarketcap==2.0.1
 
 # homeassistant.scripts.check_config
+colorama<=1
```

Comment:
This should not be getting updated. Please make sure you are on the latest dev version before running `gen_requirements_all.py`.